packages/grid/_modules_/grid/GridComponent.tsx

```diff
@@ -94,70 +95,66 @@ export const GridComponent = React.forwardRef<HTMLDivElement, GridComponentProps
     const showNoRowsOverlay = !props.loading && gridState.rows.totalRowCount === 0;
     return (
       <ApiContext.Provider value={apiRef}>
-        <AutoSizer onResize={onResize} nonce={props.nonce}>
-          {(size: any) => (
-            <GridRoot
-              ref={handleRef}
-              className={props.className}
-              size={size}
-              header={headerRef}
-              footer={footerRef}
-            >
-              <ErrorBoundary
-                hasError={errorState != null}
-                componentProps={errorState}
-                api={apiRef!}
-                logger={logger}
-                render={(errorProps) => (
-                  <GridMainContainer>
-                    <components.ErrorOverlay
-                      {...errorProps}
-                      {...componentBaseProps}
-                      {...props.componentsProps?.errorOverlay}
-                    />
-                  </GridMainContainer>
-                )}
-              >
-                <div ref={headerRef}>
-                  <components.Header {...componentBaseProps} {...props.componentsProps?.header} />
-                </div>
+        <NoSsr>
```

Comment:
What is this component? And what does it bring?

Reply:
https://github.com/mui-org/material-ui-x/pull/930#issuecomment-768932069